src/elisp/treemacs-rendering.el

```diff
@@ -97,53 +97,53 @@ the height of treemacs' icons must be taken into account."
   "Sort F1 and F2 alphabetically asc."
   (declare (pure t) (side-effect-free t))
   (inline-letevals (f1 f2)
-    (inline-quote (string-lessp ,f2 ,f1))))
```

Comment:
Commits should be squashed, commit message should be prefixed with `[Rendering]`.

Reply:
Done.